property.go

```diff
@@ -24,13 +26,22 @@ type PropertyDefinitionId struct {
 	Aliases []string `json:"aliases,omitempty"`
 }
 
+// Property represents a custom property value assigned to an entity.
 type Property struct {
 	Definition       PropertyDefinitionId `graphql:"definition"`
+	Locked           bool                 `graphql:"locked"`
 	Owner            EntityOwnerService   `graphql:"owner"`
 	ValidationErrors []OpsLevelErrors     `graphql:"validationErrors"`
 	Value            *JsonString          `graphql:"value"`
 }
 
+// PropertyConnection represents the connection type for Property.
+type PropertyConnection struct {
```

Comment:
this doesn't appear to be used anywhere

Reply:
@rocktavious this was a mistake in my diff - removed